tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -172,11 +174,17 @@ func constructConfig() *config.Config {
 	}
 
 	// Detect whether IPv6 is enabled by checking if the pod's IP address is IPv4 or IPv6.
-	podIP, err := getLocalIP()
+	podIPs, err := getLocalIPs()
 	if err != nil {
 		panic(err)
 	}
-	cfg.EnableInboundIPv6 = podIP.To4() == nil
+
+	for _, podIP := range podIPs {
+		if netutil.IsIPv6Address(podIP.String()) {
+			cfg.EnableInboundIPv6 = true
```

Comment:
This is a big change and unrelated to the rest of the PR? can we split it out?

Reply:
I mean - this is a HUGE change, it enables dual stack for everyone that has IPv6 support, right? Which is a lot of people

Reply:
Hi @howardjohn,  this change relates to the dual stack feature, it's unavailable for IPv4 service access to IPv6 service if split this change out. 
I can call another functions `AllIPv4` and `AllIPv6` to make sure that it's dual stack environment, then set `enableInboundIPv6` as `true`

Reply:
Hi @howardjohn, I have checked some logics, and `EnableInboundIPv6` should be enabled once there is an IPv6 podIP(maybe IPv6 only or Dual Stack environment). Let's assume the scenario: an IPv4 only service access to Dual Stack service with IP families ["IPv6", "IPv4"]. This is unavailable due to there is no any iptable rule for IPv6 if `EnableInboundIPv6` is disabled.

Reply:
I am not saying we should't do this at all, but its an absolutely **massive** change to suddenly start capturing traffic we were not previously, and its inside an unrelated PR.

This needs extensive review, testing, feature flag, documentation, etc - not part of this PR.

Reply:
Hi @howardjohn, correct, need more care,  @jacob-delgado and I will focus on doc at first. Thanks john!